readme.md

```diff
@@ -221,6 +221,7 @@ echo $i_oct_heart
 | [Monoid Nerd Font][p-monoid]                      |                                   | NO   | 1536    | ![w] ![m2] ![l]   |
 | [Mononoki Nerd Font][p-mononoki]                  | [Mononoki][f-mononoki]            | NO   | 1024    | ![w] ![m2] ![l]   |
 | [M+ (MPlus) Nerd Font][p-mplus]                   |                                   | NO   | 1000    | ![w] ![m2] ![l]   |
+| [NerdPlex*][p-nerdplex]                           | [IBM Plex][f-ibm-plex]            | YES  | ????    | ![w] ![m2] ![l]   |
```

Comment:
What should I write in the "EM Size" column here?

Reply:
You can find out via FontForge or other font editor programs. Looks like it is 1000